ci/travis_install_osx.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+brew update
+brew bundle --file=$TRAVIS_BUILD_DIR/c_glib/Brewfile
```

Comment:
Shouldn't that be conditioned on ARROW_CI_C_GLIB_AFFECTED?

Reply:
Yes

Reply:
@pitrou This is already conditioned on in `.travis.yml` just before this script is called. Is it really necessary to condition on it again?

Reply:
Not really, though given the filename it might be better to avoid further mistakes :-)